content/pages/office-policies.md

```diff
@@ -0,0 +1,50 @@
+Title: Office Policies
+Category: pages
+Status: hidden
+Slug: office policies
+
+
+WiCS OFFICE POLICY (Version 2.0 March 8th 2015) 
+
+PERSONNEL:
+
+* If the office is open for office hours, drop-ins or for any other reason, a MINIMUM of ONE committee member of WiCS must be present in the office at all times. 
+
+* All WiCS members occupying the office are responsible for complying with and enforcing the Code of Conduct. 
+
+
+CLEANUP:
+
+* The last WiCS staff member to occupy the office is responsible for cleanup.
+    * If the recycling and/or garbage is full, empty it. 
+    * NO PERISHABLE FOOD ITEMS ARE TO BE LEFT IN THE OFFICE.
+    * Ensure no hardware is left on the windowsill.
+    * Ensure the chairs are stacked next to the metal cabinet.
+    * Ensure there are no stray cables on the floor. 
+    * Ensure no garbage is left on the desks, chairs, shelves or floor. 
+    * Ensure the lights are turned off.
+    * Lock the door.
+
+
+OFFICE HOURS:
+
+* Office hours and services are open only to ciswomen, transwomen and non-binary folks.
+
+* The Code of Conduct is in effect at all times during office hours 
+
+CODE OF CONDUCT:
+
+ANY PERSON WHO ENTERS THE WOMEN IN COMPUTER SCIENCE OFFICE IS SUBJECT TO THE CODE OF CONDUCT. THIS INCLUDES COMMITTEE MEMBERS.
+VIOLATIONS OF THE CODE OF CONDUCT WILL RESULT IN LOSS OF PRIVILEGES, FORMAL WARNINGS,  AND/OR LOSS OF MEMBERSHIP. 
```

Comment:
We don't have membership, so I'm not sure how people can lose their memberships? We should also probably be explicit about what privileges can be lost.